client/src/ui/atoms/select/index.tsx

```diff
@@ -17,19 +17,9 @@ type SelectProps = {
   children?: React.ReactNode;
 };
 
-export const Select = ({
-  name,
-  extraClasses,
-  id,
-  isDisabled,
-  onClickHandler,
-  onFocusHandler,
-  size,
-  state,
```

Comment:
Ditto for these removed props.

Reply:
It appears that this whole atoms/select component is not used anywhere:

<img width="270" alt="image" src="https://user-images.githubusercontent.com/495429/160602917-5f86b1ad-27ca-40cd-b514-532c075300a0.png">

Not sure if we really want to keep it? (We probably want to keep the `index.scss` though.)

Reply:
Nope, let’s remove it. We can easily get it back should we need it.